R/ab0.R

```diff
@@ -1,19 +1,26 @@
 # uses r > 1 scale for shape
-ab0 <- function(
-    t,
-    curve_params) {
+ab0 <- function(t, curve_params) {
   y0 <- curve_params[["y0"]]
   y1 <- curve_params[["y1"]]
   t1 <- curve_params[["t1"]]
   alpha <- curve_params[["alpha"]]
-  shape <- curve_params[["r"]]
+  r <- curve_params[["r"]]
 
   beta <- bt(y0, y1, t1)
 
   yt <- 0
 
   yt_phase_1 <- y0 * exp(beta * t)
-  yt_phase_2 <- (y1^(1 - shape) - (1 - shape) * alpha * (t - t1))^(1 / (1 - shape))
+  if (r == 1) {
```

Comment:
if r == 1, the usual expression for this function reduces to `0^Inf = 0`, which isn't correct; as r -> 1, the function should reduce to the exponential.